composer.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "sulu/sulu-phpcr-migration-bundle",
+  "license": "MIT",
+  "description": "The bundle provides command to migrate the Sulu database from phpCr to the SuluContentBundle.",
+  "keywords": ["sulu", "phpcr", "migration", "content-bundle"],
+  "type": "sulu-bundle",
+  "authors": [
+    {
+      "name": "Sulu Community",
+      "homepage": "https://github.com/sulu/SuluPHPCRMigrationBundle/graphs/contributors"
+    }
+  ],
+  "require": {
+    "php": "^8.1",
+    "jackalope/jackalope-doctrine-dbal": "^1.7",
+    "jackalope/jackalope-jackrabbit": "^1.4",
+    "symfony/console": "^6.0 | ^7.0",
+    "symfony/dependency-injection": "^6.0 | ^7.0",
+    "symfony/config": "^6.0 | ^7.0",
+    "symfony/framework-bundle": "^6.0 | ^7.0"
+  },
+  "autoload": {
+    "psr-4": {
+      "Sulu\\Bundle\\PhpcrMigrationBundle\\": "src/"
+    }
+  },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^3.41",
+    "jangregor/phpstan-prophecy": "^1.0",
+    "phpspec/prophecy-phpunit": "^2.1",
+    "phpstan/extension-installer": "^1.3",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-doctrine": "^1.3",
+    "phpstan/phpstan-phpunit": "^1.3",
+    "phpstan/phpstan-symfony": "^1.3",
+    "phpstan/phpstan-webmozart-assert": "^1.2",
+    "thecodingmachine/phpstan-strict-rules": "^1.0",
+    "phpunit/phpunit": "^10.0",
+    "symfony/dotenv": "^6.0 || ^7.0",
+    "symfony/error-handler": "^6.0 || ^7.0"
+  },
+  "scripts": {
+    "php-cs-fix": "@php vendor/bin/php-cs-fixer fix",
+    "lint-php-cs": "@php vendor/bin/php-cs-fixer fix --verbose --diff --dry-run",
+    "phpstan": [
+      "@php Tests/Application/bin/console.php cache:warmup --env dev",
+      "@php vendor/bin/phpstan analyse"
+    ]
```

Comment:
add lint task and CI scripts see also: https://github.com/sulu/SuluContentBundle/blob/c890e28510896e7a8c612de4d68d0aebb2513b0d/composer.json#L99